modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,55 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory pages in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which updates the {op-system} kernel. You can create a custom machine config pool to add 64k pages to your nodes. 
+
+[IMPORTANT]
+====
+Using 64k pages is exclusive to 64-bit ARM architecture compute nodes or clusters installed on 64-bit ARM machines. If you create a machine pool on a 64-bit x86 built cluster, the machine pool and operator will degrade.
+====
+
+.Prerequisites: 
+* You installed the OpenShift CLI `oc`
+* You created a cluster with compute nodes of different architecture on one of the supported platforms
+
+.Procedure: 
+
+. Create the machine config pool on your compute node by running the following 
+command:
++
+[source,terminal]
+----
+$ oc create -f <filename>.yaml
+----
+
+. Edit the machine config pool and add the `64k-pages` parameter to the configuration:
```

Comment:
we need to recommend that the user creates a separate machine config pool for arm64 nodes and apply the config to the pool. [This ](https://access.redhat.com/solutions/5688941)article details how to create a machineconfigpool

Reply:
Also, we can use the procedure at https://docs.openshift.com/container-platform/4.13/machine_management/creating-infrastructure-machinesets.html#creating-infra-machines_creating-infrastructure-machinesets, except the optional point (5) that does not apply for us.


Example MachineConfigPool that fits our case:

```yaml
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfigPool
metadata:
  name: worker-64k-pages
spec:
  machineConfigSelector:
    matchExpressions:
      - key: machineconfiguration.openshift.io/role
        operator: In
        values:
        - worker
        - worker-64k-pages
  nodeSelector:
    matchLabels:
      node-role.kubernetes.io/worker-64k-pages: ""
      kubernetes.io/arch: arm64
```